src/components/OurStory.tsx

```diff
@@ -0,0 +1,21 @@
+
+
+const OurStory = () => {
+
+    const headText = "Our Story"
+    const infoText = "HARBS is a café born in Japan, pursuing freshness and hand-made quality above all else. Ever since we started, our one and only wish has been to make our customers say “DELICIOUS!” from the bottom of their hearts. We hope you savor every moment of joy you experience here at HARBS."
+
+    return (
+    <div className="flex flex-wrap items-start flex-col bg-harbs-yellow text-center">
+        <div className="mx-10 my-10">
+            <h2 className="text-6xl font-bold text-harbs-brown font-cormorant-garamond">{headText}</h2>
+        </div>
+       <div className="text-center mx-10">
+        <p className="text-center text-1xl font-taviraj text-harbs-brown">{infoText}</p>
```

Comment:
on wide screens the text goes way too horizontal. play w the px (padding x) for different break points.

Additionally, the actually text you may want to have line breaks in, and then you can look at the css options for white space to see what works best with it. (this was something i did with my portfolio)
https://developer.mozilla.org/en-US/docs/Web/CSS/white-space

Reply:
pic of how it looks on my laptop->  <img width="1712" alt="Screenshot 2023-04-10 at 2 38 59 PM" src="https://user-images.githubusercontent.com/71895647/230971385-acb5c85e-c331-4ad5-ab48-b4afa1166d79.png">